packages/locale/lang/ms.ts

```diff
@@ -0,0 +1,184 @@
+      month11: 'November',
+      month12: 'Disember',
+      week: 'minggu',
+      weeks: {
+        sun: 'Sun', // to be translated
+        mon: 'Mon',
+        tue: 'Tue',
+        wed: 'Wed',
+        thu: 'Thu',
+        fri: 'Fri',
+        sat: 'Sat',
+      },
+      weeksFull: {
+        sun: 'Ahad',
+        mon: 'Isnin',
+        tue: 'Selasa',
+        wed: 'Rabu',
+        thu: 'Khamis',
+        fri: 'Jumaat',
+        sat: 'Sabtu',
+      },
+      months: {
+        jan: 'Jan',
+        feb: 'Feb',
+        mar: 'Mac',
+        apr: 'Apr',
+        may: 'Mei',
+        jun: 'Jun',
+        jul: 'Jul',
+        aug: 'Aug',
+        sep: 'Sep',
+        oct: 'Okt',
+        nov: 'Nov',
+        dec: 'Dis',
+      },
+    },
+    inputNumber: {
+      decrease: 'mengurangkan',
+      increase: 'meningkatkan',
+    },
+    select: {
+      loading: 'Memuatkan',
+      noMatch: 'Tiada data yang sepadan',
+      noData: 'Tiada data',
+      placeholder: 'Pilih',
+    },
+    mention: {
+      loading: 'Memuatkan',
+    },
+    dropdown: {
+      toggleDropdown: 'Togol Dropdown',
+    },
+    cascader: {
+      noMatch: 'Tiada data yang sepadan',
+      loading: 'Memuatkan',
+      placeholder: 'Pilih',
+      noData: 'Tiada data',
+    },
+    pagination: {
+      goto: 'Pergi ke',
+      pagesize: '/halaman',
+      total: 'Jumlah {total}',
+      pageClassifier: '',
+      page: 'Halaman',
+      prev: 'Halaman sebelumnya',
+      next: 'Halaman seterusnya',
+      currentPage: 'halaman {pager}',
+      prevPages: 'Halaman {pager} sebelumnya',
+      nextPages: 'Halaman {pager} seterusnya',
+      deprecationWarning:
+        'Penggunaan yang ditamatkan dikesan, sila rujuk dokumentasi el-pagination untuk butiran lanjut',
+    },
+    dialog: {
+      close: 'Tutup dialog ini',
+    },
+    drawer: {
+      close: 'Tutup dialog ini',
+    },
+    messagebox: {
+      title: 'Mesej',
+      confirm: 'OK',
+      Batal: 'Dibatalkan',
+      error: 'Input haram',
+      close: 'Tutup dialog ini',
+    },
+    upload: {
+      deleteTip: 'tekan padam untuk mengalih keluar',
+      delete: 'Padam',
+      preview: 'Pratonton',
+      continue: 'Teruskan',
+    },
+    slider: {
+      defaultLabel: 'peluncur antara {min} dan {maks}',
+      defaultRangeStartLabel: 'pilih nilai mula',
+      defaultRangeEndLabel: 'pilih nilai akhir',
+    },
+    table: {
+      emptyText: 'Tiada Data',
+      confirmFilter: 'OK', // to be translated
+      resetFilter: 'Reset', // to be translated
+      clearFilter: 'Semua',
+      sumText: 'Jumlah',
+    },
+    tour: {
+      next: 'Seterusnya',
+      previous: 'Sebelumnya',
+      finish: 'Selesai',
+    },
+    tree: {
+      emptyText: 'Tiada Data',
+    },
+    transfer: {
+      noMatch: 'Tiada data yang sepadan',
+      noData: 'Tiada Data',
+      titles: ['Senarai 1', 'Senarai 2'],
+      filterPlaceholder: 'Masukkan kata kunci',
+      noCheckedFormat: '{total} item', // to be translated
```

Comment:
Can you also translate the rest of the items?

Reply:
Have been modified